backend/lib/services/terminals/index.js

```diff
@@ -574,6 +584,10 @@ async function getOrCreateTerminalSession ({ user, namespace, name, target, body
       .catch(_.noop) // ignore error
   }
 
+  if (target === TargetEnum.GARDEN && !isAdmin) {
+    ensureServiceAccountCleanup(client, { terminal, namespace, name: DASHBOARD_WEBTERMINAL_NAME })
```

Comment:
You do not await this async function here. But you also do not handle errores inside the function. Something must be wrong.

Reply:
good catch! await was missing
✅